### PR TITLE
{tools}[foss/2017b] udocker 1.1.1 w/ Python 2.7.14

### PR DESCRIPTION
**Pull request by A1ve5**
Still not working but a basis for an udocker easyconfig

### CODE REVIEW

File: easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+source_urls = [('https://github.com/indigo-dc/udocker/archive/v%(version)s.tar.gz', 'download')]
+sources = [SOURCE_TAR_GZ]
+
+use_pip = True
+
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb]: just to report that this easyconfig is about ready to work. module its the sanitycheck, via `eb --from-pr 5770 -r`; 

I only had to resort to this trick to let it pass (I can feel the hot breath of @boegel on my neck over this hack :) 
* `options = {'modulename': 'string'}`

then we are able to obtain (@A1ve5 : could you implement that `--version`? it's a nice to have):
```
[fgeorgatos@somewhere ~]$ ml !$;!!
ml udocker;which udocker
~/.local/easybuild/software/udocker/1.1.1-foss-2017b-Python-2.7.14/bin/udocker
[fgeorgatos@somewhere ~]$ udocker --version
Error: parsing command line, use: udocker help
```

seriously, this requires minimal wrap-up effort to get it into next release... thanks @A1ve5

**Comment by fgeorgatos** [comment]
@boegel, @all: 
can someone help @A1ve5 with the styling aspects? (I'm about out of tune with the process)

File: easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb (hunk 1 above)
Comment (A1ve5) [reply easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb]: thanks for this @fgeorgatos 
On udocker we try to mimic docker command syntax, so "udocker version" should work.
Nevertheless I'll try to cover the "--version" option as well.

Comment (A1ve5) [reply easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb]: On devel branch now: https://github.com/indigo-dc/udocker/commit/083b727aec536fee5b49201862aefa6a54dc7ab7

Comment (boegel) [reply easybuild/easyconfigs/u/UDOCKER/udocker-1.1.1-foss-2017b-Python-2.7.14.eb]: @fgeorgatos We're not savages, you can bypass the `import` check using this:

```python
options = {'modulename': False}
```

;-)

**Comment by boegel** [comment]
@A1ve5 @fgeorgatos Merging https://github.com/A1ve5/easybuild-easyconfigs/pull/1 should make this good to go (installation works using that updated easyconfig).

For future PRs: please target the `develop` branch (or use `eb --new-pr` which will do this for you), I changed the target branch to `develop` myself for this PR.

**Comment by boegel** [comment]
@A1ve5 ping on merging https://github.com/A1ve5/easybuild-easyconfigs/pull/1 to make this ready to go?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2644.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5f26cbe6d6723e4a90077095cf1e6624 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @A1ve5!

**Pull request merged**